backend/lib/rpc-service/service.js

```diff
@@ -36,7 +36,7 @@ export const rpcRequest = async (method, params) => {
       throw error
     }
   } catch (error) {
-    error.message = `Failed to make RPC request ${method}: ${error.message}`
+    error.message = `Failed to make RPC request ${method}.\n Request was: ${JSON.stringify(reqBody)}.\n Error message is: ${error.message}`
```

Comment:
A suggestion on how to shorten the message; feel free to ignore it.

```suggestion
    error.message = `Failed to make RPC request ${method}: ${error.message}`\nRequest was: ${JSON.stringify(reqBody)}.` 
```